modules/bwa.nf

```diff
@@ -0,0 +1,56 @@
+process bwa_index {
+  label 'bwa'
+  
+  input:
+    path(fasta)
+  
+  output:
+    path(bwa) , emit: index
+  
+  script:
+  """
+  mkdir bwa
+  bwa \\
+    index \\
+    -p bwa/db \\
+    $fasta
+  """
+  
+  stub:
+  """
+  mkdir bwa
+  
+  touch bwa/db.amb
```

Comment:
You don't need to change it, but if you want to save some keystrokes you could write:
```
touch bwa/db.{amb,ann,bwt,pac,sa}
```